[docs] Add example for the behat generators tool (#1013)
moodle/devdocs: general/development/tools/generator.md

```bash
php admin/tool/generator/cli/runtestscenario.php --feature=/path/to/some/testing/scenario.feature
```

:::tip Example

> An example Behat generator file can be found in the directory [`admin/tool/generator/tests/fixtures/`](https://github.com/moodle/moodle/blob/main/admin/tool/generator/tests/fixtures/). For instance, [`testscenario/scenario.feature`](https://github.com/moodle/moodle/blob/main/admin/tool/generator/tests/fixtures/testscenario/scenario.feature) creates a course with two assignment activities and enrolls one teacher along with five students.

Reviewer: ```suggestion
Some example Behat generator files can be found in [`admin/tool/generator/tests/fixtures/`](https://github.com/moodle/moodle/blob/main/admin/tool/generator/tests/fixtures/). For instance, [`testscenario/scenario.feature`](https://github.com/moodle/moodle/blob/main/admin/tool/generator/tests/fixtures/testscenario/scenario.feature) creates a course with two assignment activities and enrols one teacher along with five students.
```
